### PR TITLE
Add steal_operations metric

### PR DESCRIPTION
**Pull request by jschwe**
Follow-up to https://github.com/tokio-rs/tokio-metrics/pull/35.

The new metric is available starting with tokio v1.25.0.

Would it be okay to just bump the required tokio version and release a new tokio-metrics version? Or would you prefer something more advanced like version detection in a build script?

Edit: A different PR bumped the required tokio version already. Rebased.

### CODE REVIEW

**Review by Noah-Kennedy — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -31,7 +31,7 @@ futures = "0.3.21"
 num_cpus = "1.13.1"
 serde = { version = "1.0.136", features = ["derive"] }
 serde_json = "1.0.79"
-tokio = { version = "1.15.0", features = ["full", "rt", "time", "macros", "test-util"] }
+tokio = { version = "1.26.0", features = ["full", "rt", "time", "macros", "test-util"] }
```
Comment (Noah-Kennedy) [inline Cargo.toml:34]: Not sure why this change is here. This was already set. I wonder if GitHub's viewer is bugged?

Comment (jschwe) [reply Cargo.toml:34]: This one was for the dev-dependency.

Comment (Noah-Kennedy) [reply Cargo.toml:34]: Ah
